Source/CarthageKit/Project.swift

```diff
@@ -250,11 +250,11 @@ public final class Project {
 	}
 
 	/// Produces the sub dependencies of the given dependency
-	func dependenciesForDependency(dependency: Dependency<PinnedVersion>) -> SignalProducer<Set<ProjectIdentifier>, CarthageError> {
-		return self.cartfileForDependency(dependency)
-			.map { (cartfile: Cartfile) -> Set<ProjectIdentifier> in
-				return Set(cartfile.dependencies.map { $0.project })
-			}
+	func dependencyProjectsForDependency(dependency: Dependency<PinnedVersion>) -> SignalProducer<Set<ProjectIdentifier>, CarthageError> {
```

Comment:
This should  be named as `dependencyProjects(for dependency: Dependency<PinnedVersion>)`